midealocal/devices/__init__.py

```diff
@@ -126,7 +127,7 @@ def device_selector(
     model: str,
     subtype: int,
     customize: str,
-) -> Any:
+) -> MideaDevice:

```

Comment:
Maybe we should deprecate this method. I don't like this dynamic importing. What is you thought about this?

Reply:
I agree, would you please take care of it ?